### PR TITLE
Fixed error string when failing to import an exception

### PR DESCRIPTION
**Pull request by alex**
Thank you for contributing to PyO3!

### CODE REVIEW

File: src/exceptions.rs [hunk 1]
```diff
@@ -123,7 +123,7 @@ macro_rules! import_exception {
                                 ::std::panic!("Can not import module {}: {}\n{}", stringify!($module), err, traceback);
                             });
                         let cls = imp.getattr(stringify!($name)).expect(concat!(
-                            "Can not load exception class: {}.{}",
```
Comment (davidhewitt) [inline src/exceptions.rs:126]: TBH I would totally be happy to see "Fixed" entry in the changelog for this :)

**Review by davidhewitt — APPROVED** [review]
Thanks!

File: src/exceptions.rs (hunk 1 above)
Comment (alex) [reply src/exceptions.rs:126]: Ok, I can do one if you prefer.

Comment (alex) [reply src/exceptions.rs:126]: Done

**Pull request merged**